static/app/components/profiling/flamegraph/flamegraph.tsx

```diff
@@ -280,6 +280,11 @@ function Flamegraph(props: FlamegraphProps): ReactElement {
           minWidth: spanChart.minSpanDuration,
           barHeight: flamegraphTheme.SIZES.SPANS_BAR_HEIGHT,
           depthOffset: flamegraphTheme.SIZES.SPANS_DEPTH_OFFSET,
+          configSpaceTransform:
+            // When a standalone axis is selected, the spans need to be relative to profile start time
+            xAxis === 'standalone'
+              ? new Rect(-flamegraph.profile.startedAt, 0, 0, 0)
+              : Rect.Empty(),
```

Comment:
Should this be `undefined`? This is relying on the `||` fallback inside of `CanvasView` to mean an identity transform
```suggestion
              : undefined,
```

Reply:
Good find,I'll also do it above on the flamegraph view